packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/SuccessDialog.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Plan } from '../../../store/types';
+import { metadataFromPlan } from '../../../store/utils';
+import DialogMessage from '../../../components/DialogMessage';
+import fpnImage from '../../../images/fpn';
+
+export default ({ plan, onDismiss }: { plan: Plan; onDismiss: () => void }) => {
+  const { product_name: productName } = plan;
+  const { webIconURL } = metadataFromPlan(plan);
+  return (
+    <DialogMessage onDismiss={onDismiss} data-testid="reactivate-subscription-success-dialog">
+      <img
+        alt={productName}
```

Comment:
Where the product image gets used in the subscription redirect - visible in local storybook here:

http://localhost:6006/?path=/story/routes-subscriptions--reactivation-confirmation